### ISSUE TITLE
runtime: flaky corruption on openbsd

### ISSUE THREAD

**Issue by rsc** [golang/go#55161]
```
#!watchflakes
post <- goos == "openbsd" && (`runtime.throw` || `Test ".*" ran over .* limit` || `panic: test timed out` || `fatal error: all goroutines are asleep`) && !(log ~ `unexpected fault address 0x349dc4`) && date < "2023-08-01"
```

OpenBSD has been seeing a steady stream of what looks like memory corruption resulting in runtime throws.

**Comment by rsc** [comment]
Updated pattern to match panic test timed out.

**Comment by mknyszek** [comment]
Because #34988 is still open these memory corruption issues are difficult to detangle from these being real runtime issues (vs. fork corruption).

@golang/openbsd Can you please take a look at these and let us know what you think?

**Comment by bcmills** [comment]
https://storage.googleapis.com/go-build-log/e70a4d88/openbsd-amd64-70_61e058fc.log (a TryBot on https://go.dev/cl/451218)

**Comment by bcmills** [comment]
@golang/openbsd, is anyone looking into this issue? It seems to affect all OpenBSD builders and TryBots to at least some extent.

**Comment by bcmills** [comment]
@qbit, see https://cs.opensource.google/go/x/build/+/master:env/openbsd-amd64/README. You'll likely need someone on @golang/release to help with deployment.

**Comment by bcmills** [comment]
(The last OpenBSD builder upgrade was #48977.)

**Comment by bcmills** [comment]
The failures on the `-71` builders appear to be mostly (or entirely?) deadlocks, tracked separately in #57585.

**Comment by bcmills** [comment]
@4a6f656c, `openbsd-arm64-jsing` is the only once in that recent update that is not explained by #57585.

**Comment by bcmills** [comment]
- The failures with `fatal error: all goroutines are asleep - deadlock!` are #61454.
- The timeout in `context.ExampleAfterFunc_cond` involves a condition variable, so my guess is it's a bug in the test. (I will follow up with a separate issue for that.)
- The `fatal error: traceback did not unwind completely` failure mode resembles a failure on `linux-arm64-longtest` in #62105. It's not obviously specific to `openbsd`.

I'm going to close this issue as obsolete, and add a date constraint on the `watchflakes` pattern.
